### ISSUE TITLE
Purpose of normalizing dashes to "dash" in name?

### ISSUE THREAD

**Issue by apham-sy** [ACloudGuru/serverless-plugin-aws-alerts#144]
# This is a (Bug Report / Feature Proposal)
This is more of a question, and not sure if this is intended.

## Description
My api name has dashes in it, however because it's named "api-name-v1-this-is-the-function-name", the package updates the name for the cloudwatch alert to replace it as apiDashnameDashv1DashthisDashisDashtheDashfunctionDashname

Is there a way to disable this, or what is the intended reasoning to replace this as we can use dashes in the title as the prefixTemplate option can contain dashes just fine. e.g. `prefix-template-dashes-worksapiDashnameDashv1DashthisDashisDashtheDashfunctionDashname` assuming `prefixTemplate: prefix-template-dashes-work` and `apiName:api-name-v1-this-is-the-function-name`
* ***Serverless Framework Version***:
 "serverless": "^1.83.0",

**Comment by adikari** [comment]
Maybe we could add some config to disable this behaviour. Happy for you to create PR.

**Comment by gio-trueper** [comment]
Are there plans to implement this as a feature?
The default behaviour makes unnecessarily hard to parse the notifications.
